system/Debug/Toolbar.php

```diff
@@ -484,25 +484,25 @@ public function respond()
         if ($request->getGet('debugbar_time')) {
             helper('security');
 
+            //Validate and sanitize the debugbar_time parameter -- ss
+            $debugbarTime= $request->getGet('debugbar_time');
+            if (!preg_match('/^[a-zA-Z0-9_]+$/', $debugbarTime)) {
+                throw new \InvalidArgumentException('Invalid debugbar_time parameter.');
+            }
```

Comment:
Have you checked what value comes from the `debugbar_time`? I'm pretty sure we will get an exception every time.